src/phoenix/server/api/routers/v1/experiment_runs.py

```diff
@@ -7,7 +7,8 @@
 from strawberry.relay import GlobalID
 
 from phoenix.db import models
-from phoenix.experiments.types import ExperimentRun, ExperimentRunOutput
+from phoenix.db.models import ExperimentRunOutput
+from phoenix.experiments.types import ExperimentRun
```

Comment:
blurring the lines of server / client code seems kinda dangerous here as it will make decoupling them a bit more complex in the future. I get the consistency this provides but ultimately the consistency needs to come from the openAPI spec and not shared types I think.

Not blocking but I've found this pattern to ultimately just make untangling the two difficult in the long run.

Reply:
This makes sense to me. I'll clean it up. Thanks